### PR TITLE
Python project rest api

### PR DESCRIPTION
**Pull request by ghost**
** branch **
https://github.com/rumansiddiqui/insta_world_app/tree/python_project_rest_api

### CODE REVIEW

File: account/views.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (rumansiddiqui) [inline account/views.py]: Add new line at the end of the file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+venv
```
Comment (rumansiddiqui) [inline .gitignore]: Add new line at the end of the file

File: templates/signin.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (rumansiddiqui) [inline templates/signin.html]: Add new line at the end of the file

File: account/views.py (hunk 1 above)
Comment (ghost) [reply account/views.py]: resolved

File: templates/signin.html (hunk 3 above)
Comment (ghost) [reply templates/signin.html]: resolved

File: account/views.py (hunk 1 above)
Comment (ghost) [reply account/views.py]: resolved

**Pull request merged**